src/lib/services/identity.ts

```diff
@@ -12,7 +12,7 @@ const getMemberByAlias = async (alias: string) => {
   }
 
   if (res.status === 404) {
-    throw new NotFound(`Member "${alias}" does not exist`)
+    throw new NotFound('identity')
```

Comment:
is this not making the error message less helpful, before it would you tell you which alias wasn't found

Reply:
Hang on this is actually worse than this. `alias` isn't a path parameter so this should be a `400` not a `404`

Reply:
it will tell you, we just need to type less characters 
Instead of writting all message we can 'peak into' the TS interface **NotFound** and it will return the message, is just that we need to type less as it will -> 

Happy to update not found to `does not exist`
```ts

  constructor(item: Entities) {
    super({ code: 404, message: `${item} not found` })
    this.item = item
    this.name = 'not found'
    // this.stack = (<any> new Error()).stack
  }
```

Reply:
still is not a 404

Reply:
I'm just trying to make our lives easier when demoing, identity setting can be confusing and easy to forget

Reply:
point taken and address should be visible with the latest commit. Changed it to be a string.

Once we have all entities known , then we can update this to be strongly typed e.g. not just any string, but one of... 'table', 'chair', 'wall'...

At the moment it can be any string, ideally we would like to translate models into available items if it makes sense. The following PR

Reply:
This has been updated along with Entities so it's no longer an Entity - instead it can be any string - (for time being) already worried about the merge.